x/wasm/internal/keeper/test_common.go

```diff
@@ -419,36 +415,98 @@ func keyPubAddr() (crypto.PrivKey, crypto.PubKey, sdk.AccAddress) {
 	return key, pub, addr
 }
 
-var _ types.WasmerEngine = &AlwaysFailMockWasmer{}
+var _ types.WasmerEngine = &alwaysPanicMockWasmer{}
```

Comment:
Very clear name

Reply:
can't you also define

`var alwaysPanicMockWasmer = &MockWasmer{}`

And then just refer to that where needed?